packages/core/aqua-compiler/src/index.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+    generateSources,
+    generateTypes,
+} from './generate/index.js';
+import { CompilationResult } from './generate/interfaces.js';
+
+type OutputType = 'js' | 'ts';
```

Comment:
seems to be a duplicate. Can be imported

Reply:
In this case this duplication is correct and intended. Those types are not connected.

Reply:
Don't relly know why they are not connected, you didnt explain it at all. For me they look one and the same

Reply:
First type related to outermost user api. Second one intended for internal generated package